src/app/pages/instances/components/all-instances/instance-details/instance-disks/instance-disks.component.html

```diff
@@ -22,13 +30,16 @@ <h3 mat-card-title>
             ({{ instance().root_disk_io_bus | mapValue: diskIoBusLabels}})
           }
         </span>
-        <a
-          class="action"
-          ixTest="change-disk-setup"
-          (click)="showRootDiskIncreaseDialog()"
-        >
-          {{ 'Change' | translate }}
-        </a>
+
+        @if (!isVmRunning || isContainer) {
```

Comment:
Let's keep the same approach and have this and actions below as disabled instead of hiding, so that user knows that the option is there.